kfs-cam/src/main/java/edu/arizona/kfs/module/cam/document/AssetMaintainableImpl.java

```diff
@@ -0,0 +1,111 @@
+package edu.arizona.kfs.module.cam.document;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import org.kuali.kfs.sys.context.SpringContext;
+import org.kuali.rice.kns.document.MaintenanceDocument;
+import org.kuali.rice.krad.bo.Note;
+import org.kuali.rice.krad.service.KRADServiceLocator;
+import org.kuali.rice.krad.service.NoteService;
+import org.kuali.rice.krad.util.GlobalVariables;
+
+import edu.arizona.kfs.sys.KFSConstants;
+
+@SuppressWarnings("deprecation")
```

Comment:
Thanks for clarification, it helped me track down a setting issue within my own IDE (deprecation was not showing for imports in cases where the import is ultimately located in a jar). Fixed, and wow, so much faster than tracking down by hand.

I am inclined to leave this alone since the number of occurrences is greater than 2-3, and not all within one method. If, however, we really wanted to minimize suppression-scope, we could do something like:
```java
/*  
 *  Created to isolate suppression of warning to one short method, rather then entire class.
 *  Note, the preferable action would be to use the new(er) mechanism that deprecated the call
 *  here, but none was specified or found.
 */ 
@Override
@SuppressWarnings("deprecation")
public PersistableBusinessObject getBusinessObject() {
	return super.getBusinessObject();
}
```

That takes care of 6 of the 7 warnings my IDE shows after deleting the class-level @SuppressWarnings tag. The remaining warning is on line 40, reporting: "variable 'notes' initializer is redundant" (Eclipse lacks this inspection, I believe). I will address that in a separate comment since there's a question with other logic going on in the same method.